cypress/unit/store-getter-getJsonOutput.cy.js

```diff
@@ -0,0 +1,124 @@
+import { getters } from "~/store";
+
+let store = {
+
+    getters: getters,
+
+    state: {
+
+        categoricalOptions: {
+
+            "category1": [
+
+                { label: "option1Label", identifier: "option1Identifier" }
+            ],
+            "category2": [
+
+                { label: "option2Label", identifier: "option2Identifier" }
+            ]
+        },
+        categories: {
+
+            category1: {
+
+                componentName: "annot-categorical",
+                identifier: "category1Identifier"
+            },
+            category2: {
+
+                componentName: "annot-continuous",
+                identifier: "category2Identifier"
+            }
+        },
+        columnToCategoryMap: {
+
+            categoricalColumn: "category1",
+            continuousColumn: "category2",
+            uncategorizedColumn: null
+        },
+        dataDictionary: {
+
+            annotated: {
+
+                categoricalColumn: {
+
+                    missingValues: [],
+                    valueMap: { "rawValue1": "option1Identifier" }
+                },
+                continuousColumn: {
+
+                    missingValues: ["1", "2", "3"],
+                    transformationHeuristic: "transform1",
+                    UserProvidedKey: "continuousColumnUserProvidedValue"
+                },
+                uncategorizedColumn: {
+
+                    UserProvidedKey: "uncategorizedColumnUserProvidedValue"
+                }
+            }
+        },
+        transformationHeuristics: {
+
+            "transform1": {
+
+                Label: "transform1",
+                TermURL: "transform1Identifier"
+            }
+        }
+    }
+};
+
+describe("getJsonOutput", () => {
+
+    it("Output data dictionary entries for data table columns are properly formatted according to their assigned data type", () => {
```

Comment:
Suggestion: I think having one `expect` statement that checks the entirety of the `jsonOutput` improves readability here.

Reply:
Yes, I originally modeled this `expect` after your usage. But in this case, I purposely separated the cases. IMO this improves readability and more importantly the ability to separate out the tests of the differing formats across the three different case.